File: Programmers/같은숫자는싫어.py
#arr=[1,1,3,3,0,1,1]

def solution(arr):
    answer = []
    num=0
    for i in range(len(arr)):
        if num!=arr[i]:
            answer.append(arr[i])
        num=arr[i]

    return answer

print(solution(arr))

File: Programmers/같은숫자는싫어.cpp
#include <bits/stdc++.h>

using namespace std;

vector<int> solution(vector<int> arr)
{
    vector<int> answer;
    int num=-1;
    for(int i=0;i<arr.size();i++){
        if(num!=arr[i]){
            answer.push_back(arr[i]);
        }
        num=arr[i];
    }


    return answer;
}
